src/ralph/assets/api/serializers.py

```diff
@@ -322,9 +327,24 @@ class Meta:
         exclude = ('model',)
 
 
+class ProcessorSimpleSerializer(RalphAPISerializer):
+    class Meta:
+        model = Processor
+        fields = ('id', 'url', 'speed', 'cores')
+
+
+class ProcessorSerializer(ProcessorSimpleSerializer):
+    class Meta:
+        depth = 1
+        model = Processor
+        exclude = ('model',)
+
+
 # used by DataCenterAsset and VirtualServer serializers
 class NetworkComponentSerializerMixin(OwnersFromServiceEnvSerializerMixin):
     ethernet = EthernetSimpleSerializer(many=True, source='ethernet_set')
+    memory = MemorySimpleSerializer(many=True, source='memory_set')
```

Comment:
memory is used in `ComponentSerializerMixin` (which inherits from `NetworkComponentSerializerMixin`). Please move processor there as well.